tests/test_restart.cpp

```diff
@@ -52,23 +54,39 @@ BOOST_AUTO_TEST_CASE(CompareRestartFileResults)
     ecl_file_type*  file1 =  ecl_file_open_rstblock_report_step( filename1.c_str() , last_report_step, 1);
     ecl_file_type*  file2 =  ecl_file_open_rstblock_report_step( filename2.c_str() , last_report_step, 1);
 
-    for (auto key : {"PRESSURE", "SWAT", "SGAS", "RS", "RV"}) {
-        ecl_kw_type * kw_1 =  ecl_file_iget_named_kw( file1 , key, 0);
-        ecl_kw_type * kw_2 =  ecl_file_iget_named_kw( file2 , key, 0);
 
-        bool numeric_equal = ecl_kw_numeric_equal(kw_1, kw_2, abs_diff , relative_diffs[key]);
-        if (numeric_equal) {
-            std::cout << " Restart results for " << key << " compared ok" << std::endl;
+    if (!file1) {
+        std::cerr << "Failed to open restart file: " << filename1.c_str() << std::endl;
```

Comment:
Just curious about one tiny detail: Is there a reason you're using `.c_str()` here?